draft-ietf-tls-esni.md

```diff
@@ -748,7 +748,7 @@ ECHConfig, as follows:
 ~~~
     context = SetupBaseR(ClientECH.enc, skR, "tls13 ech")
     ClientHelloInner = context.Open("", ClientECH.payload)
-    ech_hrr_key = context.Export("tls13 ech hrr key", 16)
+    ech_hrr_key = context.Export("tls13 ech hrr key", 32)
```

Comment:
You also need to change this above.

s/context.Export("tls13 ech hrr key", 16)/context.Export("tls13 ech hrr key", 32)

Reply:
Grr, I thought caught them all. Thanks!